Add note about missing random generators for osx-arm64 (#3014)
nest/nest-simulator: doc/htmldoc/installation/conda_forge.rst

Conda forge install
===================

> .. note::

Reviewer: ```suggestion
.. admonition:: osx-arm64: missing random number generators
```

>    We provide conda packages for linux-64, osx-64 and osx-arm64. Due to the

Reviewer: ```suggestion
   Due to a cross-compiling issue in the conda NEST package, some random number 
   generators are not available if you are using macOS arm64 architecture. 
   The available generators are the Mersenne Twister generators `mt19937` and `mt19937_64`.
```

>    cross compiling of the conda NEST package for Apple silicon with ARM64

Reviewer: ```suggestion
```

>    architecture, there only exists a limited set of random generators.

Reviewer: ```suggestion
```
